docs/source/api/datasets.rst

```diff
@@ -120,6 +120,8 @@ Base & Utils
     :template: class.rst
 
     base.BaseDataset
+    base.BaseBIDSDataset
+    base.LocalBIDSDataset
```

Comment:
should we leave the import path of `LocalBIDSDataset` to `moabb.datasets.base` or somewhere else?

Reply:
I think we can leave it in the base, but it would be nice to be in some submodule.